### PR TITLE
Replace apt-key add with gpg --dearmor

### PR DESCRIPTION
**Pull request by jaapmarcus**
And follow the instructions as 

https://packages.icinga.com/ubuntu / debian / rasbian

And why to change it
https://blog.cloudflare.com/dont-use-apt-key/

### CODE REVIEW

**Comment by bobapple** [comment]
@cla-bot check

File: doc/02-installation.md [hunk 1]
```diff
@@ -60,14 +60,14 @@ apt-get update
...
 
 . /etc/os-release; if [ ! -z ${UBUNTU_CODENAME+x} ]; then DIST="${UBUNTU_CODENAME}"; else DIST="$(lsb_release -c| awk '{print $2}')"; fi; \
- echo "deb https://packages.icinga.com/ubuntu icinga-${DIST} main" > \
+ echo "deb [signed-by=/usr/share/keyrings/icinga-archive-keyring.gpg] https://packages.icinga.com/ubuntu icinga-${DIST} main" > \
  /etc/apt/sources.list.d/${DIST}-icinga.list
- echo "deb-src https://packages.icinga.com/ubuntu icinga-${DIST} main" >> \
+ echo "deb-src [signed-by=/usr/share/keyrings/icinga-archive-keyring.gpg] https://packages.icinga.com/ubuntu icinga-${DIST} main" >> \
  /etc/apt/sources.list.d/${DIST}-icinga.list
-
+
```
Comment (Al2Klimov) [inline doc/02-installation.md:70]: What’s this space addition for?

**Review by Al2Klimov — COMMENTED** [review]
Looks fine in general, but...

File: doc/02-installation.md (hunk 1 above)
Comment (jaapmarcus) [reply doc/02-installation.md]: It can / should be removed see last commit

**Review by Al2Klimov — APPROVED** [review]

**Comment by Al2Klimov** [comment]
* CC https://github.com/Icinga/packages.icinga.com/pull/16

**Pull request merged**